R-packages/covidcast/tests/testthat/test-covidcast.R

```diff
@@ -234,3 +236,39 @@ test_that("covidcast_days batches calls to covidcast", {
       regexp = NA)
   expect_called(m, 2)
 })
+
+# This test requires the use of stub's depth parameter, but there is a bug in
+# mockery's current release (0.4.2) which doesn't handle locked bindings
+# properly, so it errors when running devtools::check(). The test can be run
+# using testthat::test_package("covidcast"), but overwrites some functions, so
+# devtools:load_all() should be run after. This should be fixed with the next
+# release of mockery, at which point we can uncomment the test.
+# See: https://github.com/r-lib/mockery/pull/39
```

Comment:
Would you create an issue for this please so we don't forget?

Reply:
Done, #372